File: imc.js
function imc(peso, altura){
    return peso / (altura * altura);      
}

console.log("O IMC é ", imc(74, 1.75).toFixed(2));

console.log(`ba${5 * "x"}a`.toUpperCase());

console.log("Hello World".split(' '));
console.log("Hello World".split(''));

var myString = "13/01/1988";
console.log(myString.match(/\d{2}\/\d{2}\/\d{4}/));

File: imc.c
#include<stdio.h>
#include<stdlib.h>
#include<math.h>

int main(){
    
    float peso; 
    float altura; 

    printf("Digite o peso em kg do(a) aluno(a):\n");
    scanf("%f", &peso);
    printf("Digite a altura do(a) aluno(a) -> (exemplo = 1.75, para 1,75 m): \n");
    scanf("%f", &altura);
    float imc = peso / (altura * altura);
    printf("O Índice de Massa Corporal do(a) aluno(a) eh %.2f", imc);

    return 0;
}